data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/http/HttpConsumerVerticleFactory.java

```diff
@@ -238,7 +238,7 @@ private static CircuitBreakerOptions createCircuitBreakerOptions(final DataPlane
         )
         // TODO max failures should be configurable or, at least, set by the control plane
         .setMaxFailures(egressConfig.getRetry() * 2)
-        .setMaxRetries(egressConfig.getRetry())
+        .setMaxRetries(egressConfig.getRetry() + 1)
```

Comment:
I am fine in doing the `+ 1` here. But should it work without? Perhaps reasonable to track as GH issue?

Reply:
removed the +1